src/components/Inputs/ComboBox/ComboBox.tsx

```diff
@@ -64,7 +76,7 @@ export const ComboBox = <T extends ComboBoxOption<T>>(
       selected = [props.value];
     }
 
-    if (props.sortValues === false) return selected;
+    if (!sortValues) return selected;
```

Comment:
Lint warning on line 100 of ComboBox.tsx
![image](https://github.com/equinor/amplify-components/assets/17411650/c718076f-22e6-40dc-a501-7dfa3fac9eac)